### DISCUSSION TITLE
auto-start and auto-end are missing in Placement type

### DISCUSSION THREAD

**Discussion by Siregar** [gilbarbara/react-joyride#1046] [General]
Good evening! 👋
I wanted to ask why `auto-start` and `auto-end` are missing in the `placement` type of React-Joyride and React-Floater, although they are present in the type definition of Floater's README.md.

Thanks in advance!

```
type Placement = 
| "auto" | "auto-start" | "auto-end"
| "top" | "top-start" | "top-end"
| "bottom" | "bottom-start" | "bottom-end"
| "right"| "right-start" | "right-end"
| "left" | "left-start" | "left-end"
| "center"
```

**Comment by gilbarbara** [comment]
Hey @Siregar 

Those options aren't supported in react-joyride. I'll check the `react-floater` README to fix it.
